testsuite/podman_runner/18_run_secondary_parallelizable_tests_1.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
You don't need those 5 scripts with identical content, except the number.
Instead use a unique parameterizable script, where you pass that number.

Reply:
done.